vae/main.py

```diff
@@ -13,27 +13,34 @@
                     help='input batch size for training (default: 128)')
 parser.add_argument('--epochs', type=int, default=10, metavar='N',
                     help='number of epochs to train (default: 10)')
-parser.add_argument('--no-cuda', action='store_true', default=False,
```

Comment:
I think you can keep the `default=False` there and add the `default=False` to the xpu arg as well. Make the least changes to the original codes.